lib/core/test/integration/Test/Integration/Scenario/CLI/Server.hs

```diff
@@ -30,8 +36,39 @@ spec = do
                     terminateProcess ph
             threadDelay oneSecond
 
+    describe "LOGGING - cardano-wallet serve logging" $ do
+        it "LOGGING - Can log --verbose" $ \_ -> do
+            let args = ["serve", "--verbose"]
+            let process = proc' "cardano-wallet" args
+            (o, _) <- getProcStream process 1
+            o `shouldContain` "Debug"
+            o `shouldContain` "Warning"
```

Comment:
I am not sure we should see any Warning :sweat_smile:  .... I mean, we do, and that's bad but I've  talked with the node-shell team already about lowering down the severity for the iohk-monitoring-framework statistics to debug or trace. It doesn't make sense to log those as "Warnings"